### PR TITLE
Add PlaceCal training blog

### PR DESCRIPTION
**Pull request by kimadactyl**
TODO

- [x] Link sites mentioned (transdim, climate justice etc)
- [x] Re-link handbook maybe now its working
- [x] Complete PlaceCal release
- [ ] Add screenshots
- [x] Add og:photo
- [x] Reduce title length / move to subtitle
- [x] Make form a nice big CTA button ("donate" style)

### CODE REVIEW

File: content/blog/2024/placecal-organiser-training-summer-2024/index.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+We know there’s loads of things out there like this — and that none of them are working very well for the people on the ground actually doing the work. People don’t want to go and sign up for one more service, or list their thing on one more website. We agree, and have designed our approach accordingly.
+
+PlaceCal has been developed alongside an engaged community development approach that works by training up existing community organisers (like yourself!) to first go and support all the groups in your partnership to list their events using software they already have, such as Google Calendar, Outlook 365, Eventbrite, Wordpress, and SquareSpace. PlaceCal then automatically aggregates this information, requiring almost no effort once set up. This takes all the pressure off the groups to get it set up, but then lets them take it over once it’s running.
+
+Even better, once groups are in the system, they can be added to multiple partnership sites. This way, we are creating a digital commons owned by our dedicated nonprofit, building something like “wikipedia for community events”, that can be sustained past the end of any one project or volunteers’ input.
```
Comment (kimadactyl) [inline content/blog/2024/placecal-organiser-training-summer-2024/index.md:26]:
```suggestion
Even better, once groups are in the system, they can be added to multiple partnership sites. This way, we are creating a digital commons owned by our dedicated nonprofit, building something like “Wikipedia for community events”, that can be sustained past the end of any one project or volunteers’ input.
```
```suggestion
Even better, once groups are in the system, they can be added to multiple partnership sites. This way, we are creating a digital commons owned by our dedicated nonprofit, building something like `Wikipedia for community events”, that can be sustained past the end of any one project or volunteers’ input.
```

File: content/blog/2024/placecal-organiser-training-summer-2024/index.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+PlaceCal has been developed alongside an engaged community development approach that works by training up existing community organisers (like yourself!) to first go and support all the groups in your partnership to list their events using software they already have, such as Google Calendar, Outlook 365, Eventbrite, Wordpress, and SquareSpace. PlaceCal then automatically aggregates this information, requiring almost no effort once set up. This takes all the pressure off the groups to get it set up, but then lets them take it over once it’s running.
+
+Even better, once groups are in the system, they can be added to multiple partnership sites. This way, we are creating a digital commons owned by our dedicated nonprofit, building something like `Wikipedia for community events”, that can be sustained past the end of any one project or volunteers’ input.
+
+**We are still developing the tool and software but are now ready to onboard a small number of groups into our exclusive closed beta trial, 8 years in the making.**
```
Comment (kimadactyl) [inline content/blog/2024/placecal-organiser-training-summer-2024/index.md:28]:
```suggestion
**After 8 years of hard work making sure it does what community groups need, we are now ready to open it to the public. To celebrate, we are offering training on a pay-what-you-can basis for the first 10 community groups to sign up.**
```

**Comment by honor-gfsc** [comment]
- What do we want screenshots of?
- We need a new shortcode to add just a link (for the CTA button), I think. All of the existing shortcodes do specific things like mailing list signup. I don't know how to make shortcodes. 
- What do we want to use as a photo?

**Comment by kimadactyl** [comment]
> What do we want screenshots of?

Not sure if it's needed?

> We need a new shortcode to add just a link (for the CTA button), I think. All of the existing shortcodes do specific things like mailing list signup. I don't know how to make shortcodes.

Try just pasting this in to the file, if it doesn't work I'll make one properly:

```html
<aside class="kofi" markdown="0">
  <p class="kofi__text">
    Text here
  </p>
  <div class="kofi__button__container">
    <a href="https://ko-fi.com/gfscstudio" class="kofi__button" target="_blank">Donate now</a>
  </div>
</aside>
```

> What do we want to use as a photo?

Hoping rakesh can deliver or if not maybe we do need to think about stock? Or we could just use one of the illustrations?

**Comment by kimadactyl** [comment]
I am happy sans-screenshot, only last concern is if we want to move the date forward when we publish.

**Review by honor-gfsc — APPROVED** [review]

**Pull request merged**